cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -105,23 +105,27 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_
     });
   };
 
-  this.showStackTrace = (index) => {
-    //If the stack trace is showing, remove it
+  this.toggleStackTrace = (index) => {
+    //If the currently clicked row is a stack trace itself, do nothing
+    if(this.displayData[index].stackTrace && (index - 1) > 0){
```

Comment:
what is the check index-1 is for? that means if the first line has stacktrace, this check will fail?